examples/sdkconfig.defaults

```diff
@@ -21,13 +21,13 @@ CONFIG_ESP_TLS_SERVER=y
 
 CONFIG_SPIRAM_SUPPORT=y
 CONFIG_ESP32_SPIRAM_SUPPORT=y
-CONFIG_ESP32S2_SPIRAM_SUPPORT=y
-CONFIG_ESP32S3_SPIRAM_SUPPORT=y
+CONFIG_ESP32S2_SPIRAM_SUPPORT=n
```

Comment:
@npmenard - A ping on this comment; I'd like to make sure my overall understanding of the approach is correct (or not!).

Reply:
I'm sorry i thought I answered it.

`CONFIG_FEATURE_CACHE_TX_BUF_BIT` is set as a side effect to enabling `CONFIG_XXXX_SPIRAM_SUPPORT` regardless of whether or not the SPIRAM is found and setup. When the wifi drivers start this will cause the driver to allocate `CONFIG_ESP32_WIFI_CACHE_TX_BUFFER_NUM` buffer (default 32 of 1691 bytes buffer) The cache is useful to handle pressure from application layer to send data over wifi. It's not documented as it falls into their proprietary wifi implementation